capstone/client/src/components/ListPage.js

```diff
@@ -125,8 +125,13 @@ class ListPage extends Component {
       });
       return;
     }
+    let redirectAddress = '/stores/?';
```

Comment:
Looks good! Wanna try using the URL API? It should be cleaner. 
https://www.valentinog.com/blog/url/

Reply:
Ok so I tried to use the URL API but because the address is /stores instead of website.com/stores it gives me an invalid URL error